test/gpu/native/studies/transpose/transpose-time.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: Wall clock time (s):, Wall clock time (s):
+files: transpose-coalesced.dat, transpose-naive.dat
+graphkeys: Coalesced, Naive
```

Comment:
Please make sure that you replace "Coalesced" with something else as I suggested below.